charts/vector/ci/serviceHeadless-disabled.yaml

```diff
@@ -0,0 +1,8 @@
+# Separately disable the Headless Service
+## Values file for testing separate enabling for service and serviceHeadless.
```

Comment:
I'm not super familiar with yaml comments, is there a difference between one # and two ## ?

Reply:
No, both are comments... I might have copied it from somewhere. 😅 It's been a while, I think it was also a title => description sort of relationship...

Reply:
Datadog's charts have just the `# title`, maybe at some point I felt like one needed more explanation and then I did that 🤷